### ISSUE TITLE
Options of the VueI18n constructor.

### ISSUE THREAD

**Issue by pronikell** [adbrosaci/vue-lang-router#7]
Hello @radek-altof. Thank you for your work.
I have a problem with custom options for vue-i18n. (ex. [DateTime](https://kazupon.github.io/vue-i18n/guide/datetime.html#datetime-localization),  [Pluralization](https://kazupon.github.io/vue-i18n/guide/pluralization.html#custom-pluralization) etc. )
They cannot be transferred to a plugin vue-i18n.

I use a workaround:
```
--- a/src/plugin/index.js
+++ b/src/plugin/index.js
@@ -105,7 +105,9 @@ LangRouter.install = function (Vue, options) {
        i18n = new VueI18n({
                locale: defaultLanguage,
                fallbackLocale: defaultLanguage,
-               messages,
+               messages: messages,
+               dateTimeFormats: options.i18nSettings.dateTimeFormats,
+               pluralizationRules: options.i18nSettings.pluralizationRules
        });
```

and router/index.js:

```
Vue.use(LangRouter, {
  defaultLanguage: 'ru',
  translations,
  localizedURLs,
  i18nSettings
})
```
**i18nSettings** - dateTimeFormats: {...}, pluralizationRules: {..}

PS I see that vue3+ssr support is appearing, it' very cool. :+1:

**Comment by radek-altof** [comment]
Hi @pronikell, thanks for pointing that out. You're right, this should be added as an option. Will take a look at it.

**Comment by radek-altof** [comment]
Solved in [v1.3.0](https://www.npmjs.com/package/vue-lang-router/v/1.3.0) for Vue 2 and [v2.1.1](https://www.npmjs.com/package/vue-lang-router/v/2.1.1) for Vue 3.
